### ISSUE TITLE
Modals for truncated text in drawing mode

### ISSUE THREAD

**Issue by piticent123** [CFCC/telestrations#7]
sometimes people like to write overly descriptive descriptions, so we need a modal for when it doesn't fit on the bottom of the screen.

**Comment by idontusenumbers** [comment]
there's got to be a way to do it without a modal; got a screenshot of the issue?

**Comment by piticent123** [comment]
true, actually now that i think about it, an expanding card would be better. i just made this issue shortly before i left work, so i didn't want to forget to change it later, regardless of the solution

**Comment by piticent123** [comment]
i would make the text box bigger, but i don't want to take up too much drawing room, especially if people are playing on their phones

**Comment by idontusenumbers** [comment]
you might be able to use CSS grid or something to shrink the picture to make room; or have a gradient/transparent thing below the text but that might be hard to read
